wurst/closures/Execute.wurst

```diff
@@ -0,0 +1,32 @@
+package Execute
+
+public interface ForForceCallback
+	function callback()
+
+force dummy = CreateForce()
+
+init
+	ForceAddPlayer(dummy, GetLocalPlayer())
```

Comment:
Can this just be `..addPlayer(localPlayer)`? If not, can you add _force API?

Reply:
Thanks. Will add _force API as well.